src/Workloads/VSInsertion/workloads.csproj

```diff
@@ -0,0 +1,531 @@
+﻿<Project>
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
+
+  <PropertyGroup>
+    <TargetFramework>$(SdkTargetFramework)</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+
+    <!-- Don't build workloads by default unless opted in.
+         For VMR build, this project only builds in the second build pass as it depends on assets from other
+         verticals that are built in the first build pass. -->
+    <BuildWorkloads Condition="'$(BuildWorkloads)' == ''">false</BuildWorkloads>
+    <BuildWorkloads Condition="'$(DotNetBuild)' == 'true' and '$(DotNetBuildPass)' == '2'">true</BuildWorkloads>
```

Comment:
Now that the SDK bundle is building the dotnet/sdk repo in more BuildPass2 verticals, restrict this to the win-x64 buildpass2 job only?